framework/doc/content/source/systems/NonlinearSystemBase.md

```diff
@@ -63,15 +63,23 @@ factor for themselves.
 The current implementation of automatic scaling in MOOSE is fairly simple and
 proceeds according to these steps:
 
-1. Compute the on-diagonal Jacobians and/or residuals for all `Kernels`, `ScalarKernels`,
+1. Compute the Jacobians and/or residuals for all `Kernels`, `ScalarKernels`,
    `NodalKernels`, and `FVKernels`
-2. Assemble the results
-3. Examine the Jacobian diagonal or residual for every row corresponding to a
+    - If `off_diagonals_in_auto_scaling` is set to `true` in the input file,
+      then we aggregate all entries in a Jacobian row into a single row value
+      taking the absolute value of any value that is submitted to the
```

Comment:
Do you add up the absolute value of all entries as the scaling factor? Or do you take the maximum absolute value?

Reply:
We add the absolute value or all entries. Taking the maximum absolute value would require more storage as I believe we discussed